LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
```

Comment:
This is part of the original license text that serves as an appendix on how to apply the license to the readers own project and should not be changed.

Reply:
Should I add a copyright header then instead of adding in here as it was done in the other public repositories like tensorflow?

Reply:
No, the Apache License is to be kept verbatim. Yes, a few files were missing a copyright header, but I'll do a follow up PR myself using some automated tools. It's not really worth a human's time to go in and fix each one by hand.

Please revert this file and I'll merge the README changes below.

Reply:
Done as suggested. Thanks for the clarifications.